website/docs/r/dashboard.html.markdown

```diff
@@ -0,0 +1,265 @@
+                 },  
+                 ...
+                 ...
+```
+This is then referenced in the `.tf` file by using a [`template_file`](https://www.terraform.io/docs/providers/template/d/file.html) data source (terraform 0.11 or earlier), or the [`templatefile`](https://www.terraform.io/docs/configuration/functions/templatefile.html) function (terraform 0.12+).
+
+`main.tf` (terraform 0.11 or earlier):
+```hcl
+data "template_file" "dash-template" {
+  template = "${file("${path.module}/dash.tpl")}"
+  vars = {
+    md_content = "Variable content here!"
+    video_link = "https://www.youtube.com/watch?v=......"
+    sub_id = data.azurerm_subscription.current.subscription_id
+  }
+}
+
+...
+
+resource "azurerm_dashboard" "my-board" {
+    name                       = "my-cool-dashboard"
+    resource_group_name        = azurerm_resource_group.my-group.name
+    location                   = azurerm_resource_group.my-group.location
+    tags = {
+        source = "terraform"
+    }
+    dashboard_properties       = data.template_file.dash-template.rendered
+}
+
+```
+
+`main.tf` (terraform 0.12+)
+```hcl
+...
+
+resource "azurerm_dashboard" "my-board" {
+    name                       = "my-cool-dashboard"
+    resource_group_name        = azurerm_resource_group.my-group.name
+    location                   = azurerm_resource_group.my-group.location
+    tags = {
+        source = "terraform"
+    }
+    dashboard_properties       = templatefile("dash.tpl", 
+                                    {
+                                        md_content = "Variable content here!",
+                                        video_link = "https://www.youtube.com/watch?v=......",
+                                        sub_id = data.azurerm_subscription.current.subscription_id 
+                                    })
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Shared Dashboard. This should be be 64 chars max, only alphanumeric and hyphens (no spaces). For a more friendly display name, add the `hidden-title` tag.
+
+* `resource_group_name` - (Required) The name of the resource group in which to
+    create the dashboard.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `dashboard_properties` - (Required) JSON data representing dashboard body. See above for details on how to obtain this from the Portal. 
+
+* `tags` - (Optional) A mapping of tags to assign to the resource.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The Dashboard ID.
+
+## Import
+
+Dashboards can be imported using the `resource id`, e.g.
+
+```shell
+terraform import azurerm_dashboard.my-board /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.Portal/dashboards/00000000-0000-0000-0000-000000000000?api-version=2015-08-01-preview
```

Comment:
we don't need the API Version here (since we parse the ID) - so we can remove this from the example:

```suggestion
terraform import azurerm_dashboard.my-board /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.Portal/dashboards/00000000-0000-0000-0000-000000000000
```